projectDocs/dev/developerGuide/developerGuide.t2t

```diff
@@ -813,10 +824,10 @@ When this is not provided, or is less than the current version of NVDA (ignoring
 --- start ---
 name = "myTestAddon"
 summary = "Cool Test Add-on"
-version = "1.0"
+version = "1.0.0"
 description = "An example add-on showing how to create add-ons!"
-author = "Michael Curran <mick@kulgan.net>"
-url = "http://www.nvda-project.org/wiki/Development"
+author = "Michael Curran <mick@example.com>"
+url = "https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/addons.md"
 docFileName = "readme.html"
 minimumNVDAVersion = "2018.1.0"
```

Comment:
According to nvaccess/addon-datastore-validation:_tests/testData/nvdaAPIVersions.json, this will not pass as a valid minimum.